doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md

```diff
@@ -155,34 +155,134 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
     * The user for Mongo and STH is `mongo` but can be changed through the CYGNUS_MONGO_USER environment variable.
     * The pass for Mongo and STH is `mongo` but can be changed through the CYGNUS_MONGO_PASS environment variable.
+    * CYGNUS_MONGO_ENABLE_ENCODING: true applies the new encoding, false applies the old encoding.
+    * CYGNUS_MONGO_ENABLE_GROUPING: true if the grouping feature is enabled for this sink, false otherwise.
+    * CYGNUS_MONGO_ENABLE_NAME_MAPPINGS: true if name mappings are enabled for this sink, false otherwise.
+    * CYGNUS_MONGO_ENABLE_LOWERCASE: true if lower case is wanted to forced in all the element names, false otherwise.
+    * CYGNUS_MONGO_DATA_MODEL: select the data_model: dm-by-service-path or dm-by-entity.
+    * CYGNUS_MONGO_ATTR_PERSISTENCE: how the attributes are stored, either per row either per column (row, column).
+    * CYGNUS_MONGO_DB_PREFIX: prefix for the MongoDB databases
+    * CYGNUS_MONGO_COLLECTION_PREFIX: prefix for the MongoDB collections
+    * CYGNUS_MONGO_BATCH_SIZE: number of notifications to be included within a processing batch.
+    * CYGNUS_MONGO_BATCH_TIMEOUT:  timeout for batch accumulation.
+    * CYGNUS_MONGO_BATCH_TTL: number of retries upon persistence error.
+    * CYGNUS_MONGO_DATA_EXPIRATION: value specified in seconds. Set to 0 if not wanting this policy
+    * CYGNUS_MONGO_COLLECTIONS_SIZE: value specified in bytes. Set to 0 if not wanting this policy. Minimum value (different than 0) is 4096 bytes
+    * CYGNUS_MONGO_MAX_DOCUMENTS: value specifies the number of documents. Set to 0 if not wanting this policy
+    * CYGNUS_MONGO_IGNORE_WHITE_SPACES: true if exclusively white space-based attribute values must be ignored, false otherwise
+    * CYGNUS_STH_ENABLE_ENCODING: true applies the new encoding, false applies the old encoding.
+    * CYGNUS_STH_ENABLE_GROUPING: true if the grouping feature is enabled for this sink, false otherwise.
+    * CYGNUS_STH_ENABLE_NAME_MAPPINGS: true if name mappings are enabled for this sink, false otherwise.
+    * CYGNUS_STH_ENABLE_LOWERCASE: true if lower case is wanted to forced in all the element names, false otherwise.
+    * CYGNUS_STH_DATA_MODEL: select the data_model: dm-by-service-path or dm-by-entity.
+    * CYGNUS_STH_DB_PREFIX: prefix for the MongoDB databases
+    * CYGNUS_STH_COLLECTION_PREFIX: prefix for the MongoDB collections
+    * CYGNUS_STH_RESOLUTIONS: accepted values are month, day, hour, minute and second separated by comma
+    * CYGNUS_STH_BATCH_SIZE: number of notifications to be included within a processing batch.
+    * CYGNUS_STH_BATCH_TIMEOUT: timeout for batch accumulation.
+    * CYGNUS_STH_BATCH_TTL: number of retries upon persistence error.
+    * CYGNUS_STH_DATA_EXPIRATION: value specified in seconds. Set to 0 if not wanting this policy
+    * CYGNUS_STH_COLLECTIONS_SIZE: value specified in bytes. Set to 0 if not wanting this policy. Minimum value (different than 0) is 4096 bytes
```

Comment:
Wondring about "paralellism", just to confirm... CYGNUS_STH_MAX_DOCUMENTS does exist?

Reply:
A bit off topic but, internally, how Cygnus process CYGNUS_STH_COLLECTIONS_SIZE? Does also use `.capped(true)` (or somethingl like that) at collection creation point?

Reply:
CYGNUS_STH_MAX_DOCUMENTS and CYGNUS_STH_COLLECTIONS_SIZE do not exists in STH

Reply:
Makes sense. I guess that when STH sink is used then data is propagated to STH using some consumption API in that component, so the capping of collections is up to STH and Cygnus doesn't have any control.